common/app/common/pages/ImageData.java

```diff
@@ -0,0 +1,36 @@
+package common.pages;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Image;
+
+public class ImageData extends Base {
+
+    private final Image image;
+
+    private ImageData(final Image image) {
+        this.image = image;
+    }
+
+    public static ImageData of(final Image image) {
+        return new ImageData(image);
+    }
+
+    public String getThumbImage() {
+        return nullIfEmpty(image.getUrl());
+    }
+
+    public String getBigImage() {
+        return nullIfEmpty(image.getUrl());
```

Comment:
There was logic in the template
>{{#if thumbImage}}src="{{thumbImage}}"{{/if}} src="{{placeholderImage}}"

If I return an empty String this will not work.
I was satisfying that because the placeholderImage will not be in ImageData in the future but in static data. But I don't like that there is this logic in template.